Issue archiving beta (#6980)
getsentry/sentry-docs: src/includes/merge-escalating-note.mdx

<Note>

> Escalating issues does not work for merged/unmerged issues. We are working on fixing it [here](https://github.com/getsentry/snuba/issues/4558).

Reviewer: ```suggestion
Escalating issues currently does not work for merged/unmerged issues, but we're working on fixing this.
```
